client/src/components/graph/overlays/centroidLabels.js

```diff
@@ -58,13 +56,16 @@ class CentroidLabels extends PureComponent {
 
       labelSVGS.push(
         <g
+          // label is unique so disabling eslint rule
           // eslint-disable-next-line react/no-array-index-key
           key={label}
           className="centroid-label"
           transform={`translate(${coords[0]}, ${coords[1]})`}
           data-testclass="centroid-label"
           data-testid={`${label}-centroid-label`}
         >
+          {/* The mouse actions for centroid labels do not have a screen reader alternative */}
```

Comment:
why put comments in expressions - doesn't this cause runtime overhead?

Reply:
Since the comment is a child it must be wrapped in brackets or it will be converted to a react text node